ngraph/test/type_prop/convolution.cpp

```diff
@@ -14,7 +14,7 @@
 // limitations under the License.
 //*****************************************************************************
 
-#include "op/convolution.hpp"
+#include <runtime/op/convolution.hpp>
```

Comment:
`runtime/op/convolution.hp` it's not the file with references, it's Convolution v0 implementation which previously had been moved from core library to tests. 
Anyway, I'll rollback changes in the file since we not removing `runtime/op/convolution.hpp` and there is no need to remove all the type_prop tests, despite its useless right now in fact, since v0 Convolution is not in production now. Need to refactor it separately in the future.